llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -56,6 +56,7 @@ static const RISCVSupportedExtension SupportedExtensions[] = {
 
     {"smaia", {1, 0}},
     {"ssaia", {1, 0}},
+    {"smepmp", {1, 0}},
```

Comment:
This is required to be alphabetized. It won't pass lit tests if it isn't. Might even fail when tablegen runs.

Reply:
Done.